WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostDetailViewModel.kt

```diff
@@ -199,16 +229,54 @@ class ReaderPostDetailViewModel @Inject constructor(
         )
     }
 
+    private fun updateFollowButtonUiState(
```

Comment:
Question (❓): Since I didn't dig too deeply into these changes, would you mind explaining it in a sentence? From the commit message I couldn't understand the full picture in terms of how it used be behave:

```
Refactor: Update only follow button ui state to preserve related posts ui state when post is followed
```

Reply:
I made this change because related posts are fetched separately and are not part of post model. Earlier when a post was followed, `ReaderPostDetailsUiState` was recreated using the modified post model. Since modified post model. doesn't contain related posts, I chose to update just the follow button state and not recreating entire ui state.

Reply:
Thanks so much Ashita for explaining all that to me, much much appreciated! 🙏